src/Extensions/SiteTreeContentReview.php

```diff
@@ -538,6 +538,57 @@ public function canBeReviewedBy(Member $member = null)
             return false;
         }
 
+        $options = $this->getOptions();
+
+        if (!$options) {
+            return false;
+        }
+
+        if (!$options
+            // Options can be a SiteConfig with different extension applied
+            || (!$options->hasExtension(__CLASS__)
+                && !$options->hasExtension(ContentReviewDefaultSettings::class))
+        ) {
+            return false;
+        }
+
+        if ($options->OwnerGroups()->count() == 0 && $options->OwnerUsers()->count() == 0) {
+            return false;
+        }
+
+        if (!$member) {
+            return true;
+        }
+
+        // Whether or not a user is allowed to review the content of the page.
+        if ($this->owner->hasMethod("canReviewContent") && !$this->owner->canReviewContent($member)) {
+            return false;
+        }
+
+        if ($member->inGroups($options->OwnerGroups())) {
+            return true;
+        }
+
+        if ($options->OwnerUsers()->find("ID", $member->ID)) {
+            return true;
+        }
+
+        return false;
+    }
+
+    /**
+     * Check if a review is overdue and an email can be sent
+     *
+     * @param Member $member
+     *
+     * @return bool
+     */
+    public function canSendEmail(Member $member = null)
```

Comment:
All of this code is a duplication of canBeReviewedBy apart from one check so it would be better to do something like 
```
    public function canSendEmail(Member $member = null)
    {
        $canSendEmail = $this->canBeReviewedBy($member);

        if ($this->owner->obj("NextReviewDate")->InFuture()) {
            $canSendEmail = false;
        }

        return $canSendEmail;
    }
```